h2o-core/src/main/java/water/rapids/ast/prims/filters/dropduplicates/KeepOrder.java

```diff
@@ -0,0 +1,9 @@
+package water.rapids.ast.prims.filters.dropduplicates;
+
+/**
+ * Determines which duplicated row is kept during row de-duplication process.
+ */
+public enum KeepOrder {
+  First, // Retain first, drop last
```

Comment:
knit: should be "Retain first, drop rest" and "Retain last, drop rest"
... there used to be a long comment here about sort orders and such things, until I read the rest of the code and found out that its all pretty clear ...

Reply:
Re-worded. Thank you.